src/modules/launcher/Wox/MainWindow.xaml.cs

```diff
@@ -125,7 +125,7 @@ private void OnPreviewMouseButtonDown(object sender, MouseButtonEventArgs e)
                     }
                     else if (e.ChangedButton == MouseButton.Right)
                     {
-                        _viewModel.LoadContextMenuCommand.Execute(null);
+                        //_viewModel.LoadContextMenuCommand.Execute(null);
```

Comment:
Delete.

Reply:
This might still be required. I missed uncommenting this since it's part of older wox UI which I sometime use to debug. We can remove this section of code once we are ready to merge back to `dev/PowerLauncher`. Does this sound okay ?